pylab/core/models.py

```diff
@@ -56,3 +56,23 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('event-details', args=[self.starts.year, self.starts.month, self.starts.day, self.slug])
+
+
+class Attendance(models.Model):
+    NO = 2
+    YES = 1
+    MAYBE = 0
+    ATTENDANCE_CHOICES = (
+        (NO, _('No')),
+        (YES, _('Yes')),
+        (MAYBE, _('Maybe'))
+    )
+    attendee = models.ForeignKey(User)
+    event = models.ForeignKey(Event)
+    response = models.PositiveSmallIntegerField(choices=ATTENDANCE_CHOICES, default=0)
+
```

Comment:
These fields are filed automatically, one stores when object was created and other when was it was last modified. These fields are for logs and makes easier to debug, when things go wrong. You can add them like this: 

``` 
from django_extensions.db.fields import CreationDateTimeField, ModificationDateTimeField   

created = CreationDateTimeField()
modified = ModificationDateTimeField()
```